indico/modules/events/management/templates/_base_person.html

```diff
@@ -0,0 +1,51 @@
+<div class="i-form no-max-width event-person-list js-event-person-list">
+    <div class="flashed-messages"></div>
+    <div>
+        <div class="toolbar space-after">
+            <a class="icon-checkbox-checked i-button arrow js-dropdown" href="#" data-toggle="dropdown"></a>
+            <ul class="dropdown">
+                <li>
+                    <a href="#" data-select-all="#person-list-table input.select-row:visible">
+                        {% trans %}All{% endtrans %}
```

Comment:
It is more important to remove the whitespace that is enclosed inside an element (like in this case) than to remove it from cases like above
``` python
 +    {%- if is_submitter %}
 +        {% set submitter_title = _("This person submitted an abstract") %}
 +    {%- else -%}
 +        {% set submitter_title = _("This person hasn't submitted any abstracts") %}
 +    {% endif -%}
```